examples/rbd/csi-vaulttokenreview-rbac.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: v1
```

Comment:
can we create a new folder called `kms/valut` and move these deployment files related to vault to it?

Reply:
done